expense_manager/expense_manager/settings.py

```diff
@@ -54,7 +55,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [BASE_DIR / 'templates'],
```

Comment:
according to commit files , config line `[BASE_DIR / 'templates']` your PR doesn't include any 'template' folder. So  or you 
1) comment this line and set `[]` empty list 
or 
2) add template dir. 
This  decision is up to you

Reply:
i deciede 1 variant. set [] empty list. this line will be added when templates will maked